lib/tapioca/gem/pipeline.rb

```diff
@@ -228,9 +229,9 @@ def compile_object(name, value)
         @root << node
       end
 
-      sig { params(name: String, constant: Module).void }
-      def compile_module(name, constant)
-        return unless defined_in_gem?(constant, strict: false)
+      sig { params(name: String, constant: Module, ignore_defined_in_gem: T::Boolean).void }
+      def compile_module(name, constant, ignore_defined_in_gem: false)
+        return unless defined_in_gem?(constant, strict: false) || ignore_defined_in_gem
```

Comment:
I'm not very confident about this flag because of the way it's being passed all the way down from the event into the `compile_module` method. I'm curious what the `strict` keyword is supposed to do and if that expresses a similar concept.